recipes/glib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
```

Comment:
The point of sha256 as a variable is that it isolates everything that maintainers need to change into one section at the top.  I don't think we have an official standard for this.